src/main/resources/io/seqera/wave/build-view.hbs

```diff
@@ -117,6 +117,13 @@
                 <td><a href="{{scan_url}}">Image scan report</a></td>
             </tr>
         {{/if}}
+
+        {{#if build_success}}
+            {{#if build_conda_lock_url }}
+                <td>Conda Lockfile</td>
+                <td><a href="{{build_conda_lock_url}}">Conda Lockfile</a></td>
```

Comment:
`lockfile` is lower case. I'd actually prefer it as two words, but the [official conda docs](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-explicit-lockfiles-without-creating-an-environment) call it that, so let's stick to it for consistenct.

```suggestion
                <td>Conda lockfile</td>
                <td><a href="{{build_conda_lock_url}}">Conda lockfile</a></td>
```